client/components/product-list.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ProductListItem from './product-list-item';
+
+export default class ProductList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      products: []
+    };
+
+  }
+
+  componentDidMount() {
+    this.getProducts();
+  }
+
+  getProducts() {
+    const options = { method: 'GET' };
+    fetch('/api/products', options)
+      .then(resp => resp.json())
+      .then(resp => this.setState({ products: resp }));
+  }
+
+  render() {
+    if (!this.state.products.length) {
+      return <h1>No products available</h1>;
+    } else {
+      return (
+        <section className="row justify-content-around align-items-center">
```

Comment:
A `row` should suffice. Try adding a surrounding `container` if you want things "centered". And for each card, wrap it in the appropriate `col` class. That is the Bootstrap way.

https://getbootstrap.com/docs/4.5/layout/grid